src/services/keyboardShortcuts.ts

```diff
@@ -63,6 +64,26 @@ export function setupKeyboardShortcuts(scene: IndexScene) {
     }),
   });
 
+  // Copy time range
+  keybindings.addBinding({
+    key: 't c',
+    onTrigger: () => {
+      const timeRange = sceneGraph.getTimeRange(scene);
+      setWindowGrafanaSceneContext(timeRange);
```

Comment:
Not entirely sure what this is doing.

Reply:
you are not alone :P 
I added the docblock from the definition in scenes, the important bit `This is also used from TimeSrv to access scene time range.`